includes/ApplHandlingByID.php

```diff
@@ -151,6 +151,7 @@ function approve ( $id, $type, $user )
 			}
 			$next = explode ( ',', $status[ $flag + 1 ] );
 			$NEXT = str_replace('.','$', $next[0] );
+			$NEXT = str_replace(' ','_', $NEXT );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 0 spaces before closing bracket; 1 found
* No space found after comma in function call

<!-- Comment by [SideCI](https://sideci.com) -->